token-metadata/program/src/instruction/mod.rs

```diff
@@ -309,6 +328,7 @@ pub enum MetadataInstruction {
     #[account(6, name="token_program", desc="Token program")]
     #[account(7, name="system_program", desc="System program")]
     #[account(8, optional, name="rent", desc="Rent info")]
+    #[account(9, name="sysvar_instructions", desc="Instructions sysvar account")]
```

Comment:
Does the `metadata` account need to be writable?

Reply:
Yes, thanks!